lib/plugins/auto-load.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const _ = require('lodash');
+const debug = require('debug')('voxa');
+
+function autoLoad(skill, config) {
+  skill.onRequestStarted((alexaEvent) => {
+    if (!config) {
```

Comment:
this and all checks on the config should be right in the autoload function and not inside the onRequestStarted callbacks so it's a configuration error catched on starting the skill and not in runtime